src/chi/components/bulk-actions/bulk-actions.scss

```diff
@@ -0,0 +1,117 @@
+@import '_global-variables';
+@import '_global-mixins';
+
+.chi-bulk-actions {
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  padding: 0.5rem;
+
+  @include respond-to(md) {
+    background-color: $bulk-actions-background-color;
+    justify-content: space-between;
+    padding: 0.25rem 0.875rem 0.25rem 1.5rem;
+  }
+
+  .chi-divider {
+    &.-vertical {
+      display: none;
+
+      @include respond-to(md) {
+        border-right-color: $bulk-actions-divider-border-right-color;
+        display: block;
+        height: $bulk-actions-divider-height;
+      }
+    }
+  }
+
+  &__start,
+  &__end {
+    align-items: center;
+    display: flex;
+  }
+
+  &__end {
+    display: none;
+
+    @include respond-to(md) {
+      display: block;
+    }
+  }
+
+  &__top,
+  .chi-bulk-actions__buttons-mobile {
+    background-color: $bulk-actions-background-color;
+    display: flex;
+    left: 0;
+    position: fixed;
+    width: 100%;
+
+    @include respond-to(md) {
+      display: none;
+    }
+  }
+
+  &__top {
```

Comment:
How do we approach the scenario of using this on a page with a taller Header like this?

https://user-images.githubusercontent.com/25136131/144033343-c2318e6c-a8ba-4a2a-8fd1-0138d64ea62f.mov

Reply:
Good catch! We can discuss it with UI/UX on Wednesday meeting